models/intermediate/int_zendesk__ticket_historical_satisfaction.sql

```diff
@@ -45,7 +56,23 @@ with satisfaction_updates as (
     left join latest_score
         using(ticket_id)
 
-    group by 1, 2, 3, 4
+    left join satisfaction_metrics
+        using(ticket_id)
+
+    group by 1, 2, 3, 4, 5, 6
+
+), final as (
+    select
+        *,
+        case when lower(first_satisfaction_score) = 'good' and lower(latest_satisfaction_score) = 'bad'
+            then true
+            else false
+                end as is_good_to_bad_satisfaction_score,
```

Comment:
For is_good_to_bad_satisfaction_score and is_bad_to_good_satisfaction_score, you need to actually look at the last value, not necessary the first value. 

For example, suppose a ticket when from good -> bad -> good. The final change is considered a `is_bad_to_good_satisfaction_score`. The current logic won't take into consideration the second score.

Reply:
Ah yeah I see where the logic won't take into consideration the second score in this case. I can update this.